src/Console/NullArgumentBag.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console;
+
+use InvalidArgumentException;
+
+final class NullArgumentBag implements ArgumentBag
```

Comment:
Is this for testing? We should probably move it to `Tempest\Testing` if that's the case